oss-ui/docker-image/Dockerfile.nginx

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2018-2024 Tigera, Inc. All rights reserved.
```

Comment:
This should have the projectcalico copyright

Reply:
Yup!